### PR TITLE
Fix #28 - Upgrade to OpenAPI 3.1

### LINKED ISSUE #28: (On Hold) Upgrade to OpenAPI 3.1
https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0

Depends on:
- https://github.com/getlago/lago-openapi/issues/36

### PR DESCRIPTION
**Pull request by lorddaedra**
- [x] Upgrade OpenAPI 3.0 -> OpenAPI 3.1

Update to OpenAPI 3.1 command:
```zsh
npx openapi-format "swagger.yaml" --no-sort --convertTo 3.1 -o "openapi.yaml" && \
rm "swagger.yaml" && \
mv "openapi.yaml" "swagger.yaml"
```

Source of "add_quotes.py" file:
```python3
#!/usr/bin/env python3
import re
import sys

regex = r"^( *)type: string\n( *)examples:\n( *)- ([^'\"].+[^'\"])$"
subst = "\\1type: string\n\\2examples:\n\\3- '\\4'"

result = re.sub(regex, subst, sys.stdin.read(), 0, re.MULTILINE)
print(result)

```

Related issue: #28

### CODE REVIEW

**Comment by jdenquin** [comment]
What is the benefit of the upgrade to `3.1` ?

File: .github/workflows/editor-swagger-io.yml [hunk 1]
```diff
@@ -1,20 +0,0 @@
-name: Validate OpenAPI schema - editor.swagger.io
```
Comment (vincent-pochet) [inline .github/workflows/editor-swagger-io.yml:1]: Not sure to understand why this workflow is removed. Could you give us a bit of context?

**Review by vincent-pochet — COMMENTED** [review]
Thanks for this pull request @lorddaedra

**Review by vincent-pochet — APPROVED** [review]

File: .github/workflows/editor-swagger-io.yml (hunk 1 above)
Comment (vincent-pochet) [reply .github/workflows/editor-swagger-io.yml:1]: Thank you for those details :pray: 

Could you rebase main in your PR? Apparently their is a conflict in the swagger file

**Pull request closed without merging**